samples/cpp/tutorial_code/ImgProc/smoothing/Smoothing.cpp

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
Do not delete a file and re-add it: use git mv so that we can keep the history of the file